cohortextractor/query_engines/spark_lib.py

```diff
@@ -7,7 +7,7 @@
 otherwise be about how fragile they are.
 """
 import sqlalchemy.types
-from pyhive.sqlalchemy_hive import HiveHTTPDialect, HiveTypeCompiler
+from pyhive.sqlalchemy_hive import HiveHTTPDialect, HiveTypeCompiler  # type: ignore
```

Comment:
Is this ignore stopping us from checkout our use of these types (in which case, why?), or stopping mypy from descending into these modules (in which case could we constrain it in the config instead?).

Reply:
It's the latter, but to ignore specific lines you have to do inline comments from what I can tell.

Reply:
@ghickman I see this:

```
[mypy-foobar.*]
ignore_missing_imports = True
```

https://mypy.readthedocs.io/en/stable/running_mypy.html

Maybe that's a bit less annoying than having to add it to every import?

Reply:
Ooh brilliant. That's much nicer! I also discovered how we can do per-module overrides like this in pyproject land:

https://mypy.readthedocs.io/en/stable/config_file.html#using-a-pyproject-toml-file